src/app/components/NetworkRibbon/component/DetectionScreen.tsx

```diff
@@ -103,6 +103,7 @@ export const DetectionScreen: React.FC<DetectionScreenProps> = ({
           <button onClick={onStart} className={styles.linkButton}>
             {t(translations.wrongNetworkDialog.tutorialGuide, {
               wallet: walletName,
+              network: '',
```

Comment:
I just wanted to get rid of the console warning and since we hadn't passed any value there before, I thought this was the correct solution. But I can adjust it with a network name.